tests/testthat/test-query-join.R

```diff
@@ -16,7 +16,7 @@ test_that("generated sql doesn't change unexpectedly", {
     right = right_join(lf, lf),
     full = full_join(lf, lf)
   )
-  expect_known_output(print(reg), test_path("sql/join.sql"))
+  expect_known_output(print(reg), test_path("sql/join.txt"))
```

Comment:
I think you should leave these as is because they are not related to the current PR. 

When you're remodelling a house, you only need to bring "up to code" the bits of the house that you are touching. The same applies to refactoring.